system/Test/Fabricator.php

```diff
@@ -530,7 +531,15 @@ protected function createMock(int $count = null)
 
 		if ($this->model->useSoftDeletes)
 		{
-			$fields[$this->model->deletedField] = $datetime;
+			// Make a 20% chance of returning a deleted item
+			if (rand(1, 5) === 3)
```

Comment:
It would be nice to be able to overwrite this somehow. What if I want all my entities to have the same "status"?

Reply:
You'd use `setOverrides(['deleted_at' => date('Y-m-d H:i:s')])` or whatever. I'm open to other ideas.

Reply:
Fair enough.

Reply:
Deleted field now has null value by default 4658b613